src/main/java/org/kainos/ea/validator/AuthValidator.java

```diff
@@ -0,0 +1,26 @@
+package org.kainos.ea.validator;
+
+import org.kainos.ea.cli.LoginRequest;
+
+public class AuthValidator {
```

Comment:
This doesn't tell us what went wrong

Reply:
All Done. Was going for less information is key :(